api/src/services/import-services/observation/utils/observation-sampling-row-validator.ts

```diff
@@ -28,7 +32,8 @@ dayjs.extend(isSameOrBefore);
  */
 export function getObservationSamplingInformationRowValidator(
   samplingPeriods: SurveySamplePeriodDetails[],
-  utils: CSVConfigUtils<ObservationCSVStaticHeader>
+  utils: CSVConfigUtils<ObservationCSVStaticHeader>,
+  samplePeriodId?: number
```

Comment:
nit: add function JSDoc for this param

Reply:
Outdated. But I included Jsdoc for the new params object i'm passing now.